contracts/RewardManager.sol

```diff
@@ -178,7 +178,10 @@ contract RewardManager is Ownable, ReentrancyGuard
 
     function _availableDrawDownAmount(address _user) internal view returns (uint256) {
         uint256 currentTime = _getNow();
-        if (currentTime < startAccumulation) {
+        if(totalDrawn[_user]  == vestedAmount[_user]){
```

Comment:
please merge both statement like with `(currentTime >= endAccumulation || totalDrawn[_user]  == vestedAmount[_user])` with return as `_remainingBalance(_user);`

Reply:
@samarth30 please resolve like this , it shouldn't be AND statement